packages/stdlib-shims/stdlib-shims.0.1.0/opam

```diff
@@ -8,8 +8,9 @@ bug-reports: "https://github.com/ocaml/stdlib-shims/issues"
 tags: ["stdlib" "compatibility" "org:ocaml"]
 license: ["typeof OCaml system"]
 depends: [
+  "ocaml" {>="4.02.3"}
   "dune"
-  "ocaml" {>= "4.02.3"}
+ ("dune" {> "2.7.0"} | "ocaml" {<"4.12.0~~"})
```

Comment:
why `>` instead of `>=`? I don't think dune 2.7.0 has been released yet.

Reply:
it should be `>=` indeed! good catch